### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by marcin-pasiewicz**
Kod gotowy na produkcję.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,271 @@
 <!DOCTYPE html>
-<html>
+<html lang="pl">
 <head>
-    <title> Roboczy</title>
```
Comment (Adrianlippa) [inline index.html:4]: R U kidding me?

File: index.html [hunk 2]
```diff
@@ -1,10 +1,271 @@
 <!DOCTYPE html>
-<html>
```
Comment (Adrianlippa) [inline index.html:2]: you must be joking....

File: index.html [hunk 3]
```diff
@@ -1,10 +1,271 @@
...
+
+            <section class="functionlist">
+                <h1>
+                    <span>POWIADOM</span>
+                </h1>
+                <p>O aktualnych notowaniach i podejmij decyzję</p>
+            </section>
+
+    </article>
+    <article class="team container-fluid" id="dream_team">
```
Comment (Adrianlippa) [inline index.html:101]: this doesn't work any good

File: index.html [hunk 4]
```diff
@@ -1,10 +1,271 @@
...
+        <section class="team col-xs-12 col-sm-6">
+            <img class="img-team" src="img/bartek.jpg" alt="zdjęcie"/>
+            <h2>Bartosz Śledziński</h2>
+            <h4>"The only easy day was yesterday."</h4>
+        </section>
+
+    </article>
+
+</main>
+<div class="empty"></div>
```
Comment (Adrianlippa) [inline index.html:126]: that cross button - i approve

**Review by Adrianlippa — COMMENTED** [review]
TLDR :+1:
